### PR TITLE
[SKIP CI] Updates to README

### PR DESCRIPTION
**Pull request by tusharnt**
- Minor cleanups
- Added reference to StorageHub, Docker Store and Bintray
- Removed reference to gitter

### CODE REVIEW

**Comment by shuklanirdesh82** [comment]
Adding `[SKIP CI]` keyword to title so that our CI will not be kicked off. Please make sure to have that keyword too while merging the PR

File: README.md [hunk 1]
```diff
@@ -97,12 +96,12 @@ Please refer [vDVS configuration page](http://vmware.github.io/docker-volume-vsp
...
 **Docker (Windows)**: 1.13/17.03
 
-## Reference
+## References
 
 * **Known Issues**: Please check [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) to find out about known issues.
 
-* **Contact us**: Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html) for requesting any feature or reporting a product issue.
-
- * **Blogs**: Please check our [vDVS blog page](http://vmware.github.io/docker-volume-vsphere/documentation/blogs.html).
```
Comment (shuklanirdesh82) [inline README.md]: IMO, it is better to have reference of "Blogs" in README.md

File: README.md [hunk 2]
```diff
@@ -16,29 +15,29 @@ To read more about code development and testing please read
...
-## Detailed documentation
+## Product documentation
+1. [VMware OSS Projects Page](http://vmware.github.io/docker-volume-vsphere/documentation/).
+2. [StorageHub](https://storagehub.vmware.com/#!/vsphere-storage/project-hatchway)
 
-Detailed documentation can be found on our [GitHub Documentation Page](http://vmware.github.io/docker-volume-vsphere/documentation/).
+## Downloads
 
-## Download
+**Releases are available on [Github](https://github.com/vmware/docker-volume-vsphere/releases), [Docker store](https://store.docker.com/plugins/vsphere-docker-volume-service) and [Bintray](https://bintray.com/vmware/product/vDVS/view)**
```
Comment (shuklanirdesh82) [inline README.md]: I think we should just keep only Github releases link only as docker-store and bintray links are redundant.

File: README.md (hunk 1 above)
Comment (tusharnt) [reply README.md]: List of blogs will get outdated really fast and gives an impression that we approve everything said in those blogs so I prefer not to keep it.

File: README.md (hunk 2 above)
Comment (tusharnt) [reply README.md]: There is no place where we call out all the places customers need to go to in order to get vDVS. So I prefer to keep it.

Comment (shuklanirdesh82) [reply README.md]: We have decided to stick with "**Releases are available on [Github](https://github.com/vmware/docker-volume-vsphere/releases)**" to have central place of the information for the releases. 

Moreover, we have badge icon too for all the releases; this is totally unnecessary and redundant to me. It is good to have less verbose. 

My vote is on not adding this extra verbose. 

/CC @pdhamdhere

Comment (pdhamdhere) [reply README.md]: Yes. No point in duplicating what's already their on GitHub Releases page. I would simply redirect to "GitHub Releases" page.

File: README.md [hunk 3]
```diff
@@ -16,29 +15,29 @@ To read more about code development and testing please read
...
 
 Please check [vDVS Installation User Guide](http://vmware.github.io/docker-volume-vsphere/documentation/install.html) to get started. To ensure compatibility, make sure to use the same version of vDVS driver (on ESX) and managed plugin (on Docker host VM).
 
-## Supported Platform
+## Supported Platforms
 
 **ESXi:** 6.0 and above<br />
 **Docker (Linux):** 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)<br />
-**Docker (Windows):** 1.13/17.03 and above (Windows containers mode only)
+**Docker (Windows):** 1.13/17.03 and above (Windows containers mode only)<br />
```
Comment (pdhamdhere) [inline README.md]: I doubt if we tested with Docker 1.13.

File: README.md (hunk 1 above)
Comment (pdhamdhere) [reply README.md]: Yeah. Blogs were added in absence of documentation. Now that we have docs, let's remove blogs.

**Comment by shuklanirdesh82** [comment]
closing this PR. It is too old.

**Pull request closed without merging**